uefi/src/proto/mod.rs

```diff
@@ -81,3 +81,57 @@ pub mod shell_params;
 pub mod shim;
 pub mod string;
 pub mod tcg;
+
+/// The UEFI boot policy is a property that influences the behaviour of
+/// various UEFI functions that load files (typically UEFI images).
+///
+/// This type is not ABI compatible. On the ABI level, this is an UEFI
+/// boolean.
+#[derive(Debug, Copy, Clone, PartialOrd, PartialEq, Eq, Ord)]
+pub enum BootPolicy {
+    /// Indicates that the request originates from the boot manager, and that
+    /// the boot manager is attempting to load the provided `file_path` as a
+    /// boot selection.
+    ///
+    /// Boot selection refers to what a user has chosen in the (GUI) boot menu.
+    TRUE,
+    /// The provided `file_path` must match an exact file to be loaded.
+    FALSE,
```

Comment:
It would be nice to provide clearer names. I find the specification's text rather confusing as to exactly what the `true` value means in practice, so I'm not 100% sure of what the best names are, but how about:

`TRUE` -> `BOOT_SELECTION`
`FALSE` -> `EXACT_MATCH`

Reply:
Done in #1326